charts/vault/.gitignore

```diff
@@ -0,0 +1,2 @@
+charts
+Chart.lock
```

Comment:
why is this needed?

Reply:
Running make verify, test, etc. generated these files.